R/fb_search.R

```diff
@@ -73,6 +73,7 @@ fbad_get_search <- function(
     }
 
     if (!inherits(res, 'data.frame')) {
+        library(plyr)
```

Comment:
Instead of loading the package, we should rather import `ldply` at https://github.com/cardcorp/fbRads/blob/master/R/zzz.R#L9